src/data/csv_parser.h

```diff
@@ -79,11 +79,21 @@ ParseBlock(const char *begin,
     const char* p = lbegin;
     int column_index = 0;
     IndexType idx = 0;
-    float label = 0.0f;
+    DType label = DType(0.0f);
 
     while (p != lend) {
       char *endptr;
-      float v = strtof(p, &endptr);
+      DType v;
+      // if DType is float32
+      if (std::is_same<DType, real_t>::value) {
+        v = strtof(p, &endptr);
+      // If DType is uint32
```

Comment:
Looks like the comment needs to be updated

Reply:
Done